collectors/performance/appdynamics/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM docker.io/openjdk:8-jre
+
+MAINTAINER Hygieia@capitalone.com
+
+
+ENV SPRING_DATA_MONGODB_DATABASE=dashboard
+ENV SPRING_DATA_MONGODB_HOST=10.0.1.1
+ENV SPRING_DATA_MONGODB_PORT=9999
+ENV SPRING_DATA_MONGODB_USERNAME=db
+ENV SPRING_DATA_MONGODB_PASSWORD=ENC(aSPTk36yA/ZklUg75CrZ8w==)
```

Comment:
@virtualpeter Please take these out. The default values are set in the prop builder.

Reply:
these lines are necessary to support docker-compose v2 and help when debugging this declares the variables so they can be passed in at runtime -  i followed the pattern set by all the other collectors' Dockerfile and prop builders in that they are defaulted here as well as in the collector.

Reply:
@virtualpeter the ENV variables can be passed through docker-compose.yml file.